### PR TITLE
Update graphiql-workspace to 1.0.4

### LINKED ISSUE #37: Update for GraphiQL changes
@tlvenn Do we need to update the "advanced" GraphiQL (now GraphiQL Workspace), based on your conversation here: https://github.com/OlegIlyenko/graphql-toolbox/issues/3 ?

### PR DESCRIPTION
**Pull request by tlvenn**
Fixes #37

### CODE REVIEW

**Comment by bruce** [comment]
Holding this for now; it appears that there may be a regression with debouncing introspection queries while the user types queries.

File: lib/absinthe/plug/graphiql_workspace.html.eex [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+If you wish to receive JSON, provide the header "Accept: application/json" or
+add "&raw" to the end of the URL within a browser.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  
+  <title>GraphiQl Workspace</title>
```
Comment (jparise) [inline lib/absinthe/plug/graphiql_workspace.html.eex]: @tlvenn: While you're here, could you also fix the spelling of "GraphiQL"? 😉

Comment (tlvenn) [inline lib/absinthe/plug/graphiql_workspace.html.eex]: sure thing ;)

Comment (tlvenn) [inline lib/absinthe/plug/graphiql_workspace.html.eex]: Done.

**Comment by tlvenn** [comment]
@bruce should be good to go now.

**Comment by bruce** [comment]
@tlvenn Currently getting a 404 for https://unpkg.com/graphiql-workspace@1.0.2/graphiql-workspace.min.js

**Comment by G3z** [comment]
I can see the file you linked, is it a network problem perhaps?

**Comment by benwilson512** [comment]
Works for me too. Should we bundle the javascript ourselves? This would enable offline functionality too.

**Comment by bruce** [comment]
@G3z Possibly? I tried for several minutes, but works fine now.
@benwilson512 Possibly, but out of scope of this change.

**Comment by bruce** [comment]
@tlvenn Seems to work fine, but I'm getting this in the console:

developer_tools_-_http___localhost_4000_api_graphiql

This error does _not_ appear on http://toolbox.sangria-graphql.org/graphiql

**Comment by tlvenn** [comment]
weird I did not see that prop type issue at all when I played with it...

**Comment by tlvenn** [comment]
And my change should not be able to trigger this kind of issue at all.

**Comment by tlvenn** [comment]
Ya so save and clean workspace are borked somehow... Looking into it.

**Comment by tlvenn** [comment]
I dont know if I should cry or laugh...
graphiql_storage
graphiql_storage2

So basically someone forgot that the storage is pluggable and not necessarily `localStorage`  and should obey the defined prop types.

So the prop types need to be updated on GraphiQL side and I need to add the `remoteItem` function to the storage implementation on graphiql-workspace side.

**Comment by bruce** [comment]
@tlvenn Thanks!

**Comment by tlvenn** [comment]
Fixed with https://github.com/OlegIlyenko/graphiql-workspace/pull/15

**Comment by tlvenn** [comment]
Fixed with graphiql-workspace 1.0.4 which we use now.

**Comment by bruce** [comment]
Released in v1.2.5.

**Pull request merged**